server/data/seeds/old/006_students_classes.js

```diff
@@ -4,23 +4,22 @@ const students_classes = [];
 const s_ids = [];
 const c_ids = [];
 
-for (let i = 0; i < 100; i++) {
+for (let i = 1; i < 101; i++) {
   s_ids.push(i);
   c_ids.push(i);
 }
 
 for (let i = 0; i < 100; i++) {
+  const [class_id] = [...c_ids.splice(Math.floor(Math.random() * c_ids.length), 1)];
+  const [ student_id ] = [...s_ids.splice(Math.floor(Math.random() * s_ids.length), 1)];
```

Comment:
So the splice pulls out the object and then this `[ student_id ]` is using object destructure to pull out specifically that key in that object?

Reply:
Yeah I noticed Justin had this in his code previously but for some reason it didn't carry over on a pull from master, so I added it again. The only issue I am finding is that it does not allow a class to have multiple students or a student to have multiple classes, so that is something that will need to be adjusted.

Reply:
Haha I'm not sure why I did this. I think Math.random() would work just as well or better. I must have been thinking the student/class ids had to be unique.

Reply:
Oh okay, that makes sense! We can change it this week and see if math.random() alone would fix the issue